packages/shared/package.json

```diff
@@ -7,7 +7,7 @@
     "lint:css": "stylelint '**/*.{ts,tsx,css}'",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0",
     "pretest": "npm run lint && npm run lint:css",
-    "test": "node --expose-gc ./node_modules/.bin/jest --logHeapUsage",
+    "test": "NODE_ENV=test node --expose-gc ./node_modules/.bin/jest --logHeapUsage",
```

Comment:
I think this is set automatically by jest

Reply:
Oh, nice. I didn't know about this one. Thank you!